wordpress-mobile/WordPress-Android
# Merge EditPostPreviewFragment into PostPreviewFragment

Fixes # https://github.com/wordpress-mobile/WordPress-Android/issues/8964

Initial problem was that `EditPreviewPostFragment` wasn't using `WPWebViewClient` as a client in `WebView` which exposed an issue https://github.com/wordpress-mobile/WordPress-Android/issues/8964. (Missing logic for inserting the token when fetching private site image)

We had two very similar classes `EditPreviewPostFragment` and `PreviewPostFragment`.
After some testing and analyzing of the code from both of them, I have decided to merge them into one class `PreviewPostFragment` with cherry-picking the most suitable logic.

To test:

### Steps to reproduce the behavior

**Test A:**
1. Select a WordPress.com Simple site and set it to Private.
2. Start a new post.
3. Add an image to the post. (Wait for it to upload or select an image from the site's media library.)
4. Preview the post. Result: The local draft shows an image.
5. Back to the post, add some content
6. Preview the post. Result: It should have the latest changes
7. Rotate device. Result: Content should be there

**Test B:**
1. Select a WordPress.com Simple site and set it to Private.
2. Create a local draft with an image
3. From the post list, select "Preview" to preview the local draft with an image. Result: The remote draft loads the image as expected.
4. Rotate device. Result: Content should be there

Update release notes:
